docs/_sources/user/group_mdmr.rst

```diff
@@ -38,15 +38,10 @@ Configuration Using a YAML File
 """"""""""""""""""""""""""""""""
 To configure CWAS options within a YAML file, add the following lines to your file (with appropriate substitutions for paths):
  
-.. code-block:: yaml
-
-    runCWAS = [0]
-    cwas_roi_file = '/path/to/cwas_mask_file'
-    cwas_regressor_file = '/path/to/cwas_regressor_file'
-    cwas_regressor_participant_column = 'ID
-    cwas_regressor_columns = ['FIQ', 'VIQ', 'PIQ']
-    cwas_permutations = 500
-    cwas_parallel_nodes = 3
+.. literalinclude:: /references/group_config_template.yml
```

Comment:
I'm pretty sure this file doesn't exist yet, but if you add
```BASH
ln ${CPAC_DIRECTORY}/resources/configs/group_config_template.yml /references/group_config_template.yml || true
```
to the end of https://github.com/FCP-INDI/fcp-indi.github.io/blob/c5f97f027866bb906a2b2408b2fd0ee4cdede355/bin/build#L3-L7 the builder will hotlink the config file from the installed version of C-PAC to that path in the docs source directory

Reply:
Resolved in 387f71f8f53b24bef1479cf778b90b475b27a5ee + e18a0197b686d6d7722492ed96cd6689077b6fdc:

https://github.com/FCP-INDI/fcp-indi.github.io/blob/e18a0197b686d6d7722492ed96cd6689077b6fdc/bin/build#L8